dot/core/interface.go

```diff
@@ -66,12 +66,12 @@ type BlockState interface {
 	FreeImportedBlockNotifierChannel(ch chan *types.Block)
 	GetFinalisedNotifierChannel() chan *types.FinalisationInfo
 	FreeFinalisedNotifierChannel(ch chan *types.FinalisationInfo)
-	HighestCommonAncestor(a, b common.Hash) (common.Hash, error)
 	SubChain(start, end common.Hash) ([]common.Hash, error)
 	GetBlockBody(hash common.Hash) (*types.Body, error)
 	HandleRuntimeChanges(newState *rtstorage.TrieState, in runtime.Instance, bHash common.Hash) error
 	GetRuntime(blockHash common.Hash) (instance runtime.Instance, err error)
 	StoreRuntime(common.Hash, runtime.Instance)
+	LowestCommonAncestor(a, b common.Hash) (common.Hash, error)
```

Comment:
nit whilst we're at it, it would be nice to name the return (I guess `commonAncestorBlockHash`?)

Reply:
I think it's understood that it's the block hash.

Reply:
Meh for us sure, not for a newcomer

Reply:
Even for a newcomer, it's part of the `BlockState` interface.  I think it's pretty clear it's a block hash and not a trie node hash.